assets/js/builder/Models/Quiz.js

```diff
@@ -46,14 +48,16 @@ define( [
 		schema: QuizSchema,
 
 		/**
-		 * New lesson defaults
-		 * @return   obj
-		 * @since    3.16.0
-		 * @version  3.16.6
+		 * New lesson defaults.
+		 *
+		 * @since 3.16.0
+		 * @since [version] Added filter for filtering defaults.
+		 *
+		 * @version [version]
```

Comment:
```suggestion
		 *
		 * @return {Object}
```
as you know we don't add the version tag anymore...